RecoEgamma/EgammaTools/python/calibratedEgammas_cfi.py

```diff
@@ -0,0 +1,66 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
this file should be a cff, not a cfi because multiple producers are defined

Reply:
BTW, a large fraction of this file is redundant to the auto-generated calibratedElectronProducer_cfi, calibratedPatElectronProducer_cfi (same for the photons).
To avoid diverging definitions, please start using the files made from the fillDescriptions (see $CMSSW_BASE/cfipython/$SCRAM_ARCH/RecoEgamma/EgammaTools/)

Reply:
I wasnt entirely sure how to do this but improved it slightly with the other comments

Reply:
(the auto generated part, I of course renamed the file)